source/queue_policy/policies.rst

```diff
@@ -25,171 +25,61 @@ Login nodes should be used for tasks similar to the following:
 Use compute nodes for processes that require more cores, longer run
 times, or more memory.
 
-Cron Usage
-==========
+.. _cron_usage_policy:
 
-To schedule recurring jobs, cron is provided for users. On most
-systems, cron jobs will be started on service/login nodes; therefore
-the login node usage policy applies to cron jobs.
-
-However, on systems such as Gaea, cron is no longer permitted. Instead,
-**scrontab** is used.
-
-scrontab
-========
+Cron and scrontab usage
+=======================
 
-**Scrontab** is a Slurm-managed crontab that runs on a designated partition
-rather than the login nodes. The scrontab command is used to set, edit, and
-remove a user’s Slurm-managed crontab. A user can set the VISUAL or EDITOR
-environment variables. See ``man scrontab`` for more information.
-
-**Sample Script**
-
-.. code-block:: shell
+To schedule recurring jobs, :ref:`cron <cron_scron>` is provided for users. On
+most systems, cron jobs will be started on service or login nodes; therefore
+the login node usage policy applies to cron jobs.
 
-  #SCRON --partition=cron_c5
-  #SCRON --time=1:00:00
-  #SCRON --job-name=scron_test
-  #SCRON --chdir=/ncrc/home1/First.Last
-  #SCRON --output=/ncrc/home1/First.Last/scron_test.%j.scrontab
-  0 * * * * /ncrc/home1/First.Last/test.csh
+However, on systems such as Gaea and PPAN, cron is no longer permitted.
+Instead, you must use :ref:`scrontab <rdhpcs_scrontab>`.
 
+Any process launched from cron or scrontab must be one of the following:
 
-Jobs created with scrontab are assigned a single job id. When a job is
-cancelled, there will be no future runs, and the job will be commented out in
-the users scrontab.
+- A call to a workflow manager for controlling recurring tasks
+- other scripts that submits or manages jobs in the batch system
+- short running (less than 2 minute) data manipulation task
 
-To skip the next run of your cron job, use ``control requeue <job_id>``.
+To ensure cron and scrontab jobs do not overload the service or login nodes,
+please consider the following:
 
-Scron Expressions
+Cron/Scrontab Dos
```

Comment:
'Dos' ?  Two?  It's unclear what this means.  "Suggestions for Cron/Scrontab" perhaps?